WhaleSpotting/Models/ResponseModels/SightingResponseModel.cs

```diff
@@ -0,0 +1,53 @@
+using Newtonsoft.Json;
+using System;
+using WhaleSpotting.Models.DbModels;
+
+namespace WhaleSpotting.Models.ApiModels
+{
+    public class SightingResponseModel
+    {
+        [JsonProperty("sighted_at")]
+        public DateTime SightedAt { get; set; }
+        [JsonProperty("species")]
+        public string Species { get; set; }
+        [JsonProperty("quantity")]
+        public int? Quantity { get; set; }
+        [JsonProperty("location")]
+        public string? Location { get; set; }
+        [JsonProperty("longitude")]
+        public double Longitude { get; set; }
+        [JsonProperty("latitude")]
+        public double Latitude { get; set; }
+        [JsonProperty("description")]
+        public string? Description { get; set; }
+        [JsonProperty("orca_type")]
+        public string? OrcaType { get; set; }
+        public int? UserId { get; set; }
+        public string Username { get; set; }
+        public bool Confirmed { get; set; }
+
+        public SightingResponseModel()
+        {
+            Confirmed = true;
+            Username = "Whale Museum";
+        }
+
+        public SightingResponseModel(SightingDbModel sighting)
+        {
+            SightedAt = sighting.SightedAt;
+            Species = sighting.Species.ToString();
+            Quantity = sighting.Quantity;
+            Location = sighting.Location;
+            Longitude = sighting.Longitude;
+            Latitude = sighting.Latitude;
+            Description = sighting.Description;
+            OrcaType = sighting.OrcaType.ToString();
+            //OrcaPod = sighting.OrcaPod;
```

Comment:
Whats the reason for this being commented?

Reply:
Yes. It's just not to forget we have it in our DBmodel and not using in here. It is not in obvious why it is there as it's not in Whale Musium response

Reply:
We'll add a TODO